src/sidebars/favourites/tearout.js

```diff
@@ -11,37 +11,53 @@
                         var dragElement = element[0],
                             tearElement = dragElement.parentNode.parentNode,
                             tileWidth = tearElement.clientWidth,
-                            tileHeight = tearElement.clientHeight;
+                            tileHeight = tearElement.clientHeight,
+                            store;
 
-                        function createConfig() {
+                        function createConfig(minWidth, minHeight, width, height, url, resizable, maximizable, showTaskbarIcon, saveWindowState) {
```

Comment:
@WPFerg The thing is though it's easy to leave variables that have no sensible default out of this object. For example `minWidth`, which will be very difference for a tearout card and a main window instance. Is it better to allow the developer to (accidentally) leave this out?